.github/workflows/cml-report.yml

```diff
@@ -0,0 +1,63 @@
+name: Generate CML Report
+on:
+  pull_request:
+    # types: [ labeled ]
+    branches:
+      - master
+
+jobs:
+  generate_cml_report:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    container: 
+      image: ghcr.io/moja-global/rest_api_gcbm:master
+      ports:
+        - "8080:8080"
+    steps:
+    
+      - name: Checkout repository
+        uses: actions/checkout@v3
+          
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          
+      - name: Run the Simulation
+        run: |
+          apt-get update && apt-get install unzip -y
+          pushd local/rest_api_gcbm/tests/
+          unzip linux-demo.zip
+          cd config/
+          /opt/gcbm/moja.cli --config_file gcbm_config.cfg --config_provider provider_config.json
+          popd
+          cd local/rest_api_gcbm && mkdir GCBM.CompileResults
+          
+      - name: Checkout the Repository
+        uses: actions/checkout@v3
+        with:
+          repository: moja-global/GCBM.CompileResults
+          path: local/rest_api_gcbm/GCBM.CompileResults
+          
+      - name: Compile the Results
+        run: |
+          cd local/rest_api_gcbm
+          python3 GCBM.CompileResults/compileresults.py sqlite:///tests/output/gcbm_output.db --output_db sqlite:///tests/output/compiled_simulation_output.db
+          
+      - name: Install dependencies

```

Comment:
I think these would be fine to add to `rest_api_gcbm` too - maybe even CML while we're at it?